### PR TITLE
PXB-2752 Document azure storage option for xbcloud (2.4)

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
modified: source/index.rst
modified: source/xbcloud/xbcloud.rst
new file: source/xbcloud/xbcloud_azure.rst

### CODE REVIEW

File: storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud.rst [hunk 1]
```diff
@@ -62,7 +62,7 @@ Supported Cloud Storage Types
 
 In addition to Swift, which has been the only option for storing backups in a
 cloud storage until *Percona XtraBackup* 2.4.14, *xbcloud* supports *Amazon S3*,
-MinIO, and Google Cloud Storage. Other *Amazon S3* compatible storages, such
+MinIO, Azure, and Google Cloud Storage. Other *Amazon S3* compatible storages, such
```
Comment (patrickbirch) [inline storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud.rst]: add link to doc

File: storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud_azure.rst [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+All of the available options for **xbcloud**, such as parallel, max-retries, and others, can be used. For more information, :ref:`xbcloud_binary`.
+
+Examples
+-------------
+
+An example of an **xbcloud** backup.
+
+.. sourcecode:: bash
+
+   xtrabackup --backup --stream=xbstream --target-dir= $TARGET_DIR | xbcloud put backup_name --azure-storage-account=pxbtesting --azure-access-key=$AZURE_KEY --azure-container-name=test --storage=azure
```
Comment (patrickbirch) [inline storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud_azure.rst]: Add $ to all code samples

File: storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud.rst (hunk 1 above)
Comment (alina-derkach-oaza) [reply storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud.rst]: added

File: storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud_azure.rst (hunk 2 above)
Comment (alina-derkach-oaza) [reply storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud_azure.rst]: added

**Review by patrickbirch — APPROVED** [review]

**Comment by rahulmalik87** [comment]
LGTM

**Review by rahulmalik87 — APPROVED** [review]

**Pull request merged**